### DISCUSSION TITLE
v1.5.0

### DISCUSSION THREAD

**Discussion by ggerganov** [ggml-org/whisper.cpp#1496] [Announcements]
## [Overview](https://github.com/ggerganov/whisper.cpp/releases/tag/v1.5.0)

This major release includes the following changes:

- Full GPU processing of the Encoder and the Decoder with CUDA and Metal is now supported
- Efficient beam-search implementation via batched decoding and unified KV cache
- Full quantization support of all available `ggml` quantization types
- Support for grammar constrained sampling
- Support for Distil Whisper models
- Support for Whisper Large-v3

---

A discussion about the [whisper.cpp v1.5.0](https://github.com/ggerganov/whisper.cpp/releases/tag/v1.5.0) release.
Let us know about your experience, if you have any questions or if you find any issues

**Comment by mhubii** [comment]
1.5.0 is wild! Thank you so much for this!
